requirements/quality.in

```diff
@@ -4,9 +4,6 @@
 
 -r test.txt               # Core and testing dependencies for this package
 
```

Comment:
Nit: Personally I don't think it's better to have so many different requirements files. Just three - `base`, `dev`, and `constraints` - would cover every use case, even if `dev` will sometimes install more than is needed for a specific use case like `quality` or `ci`. But no need to change anything, just feel like mentioning it.